larray/session.py

```diff
@@ -190,6 +192,8 @@ def _dump(self, key, value, *args, **kwargs):
         self.handle[key] = value.dump(*args, **kwargs)
 
     def save(self):
+        if self.new_file:
+            self.handle['Sheet1'].delete()
```

Comment:
btw, if you want to tackle this, it would be better to avoid creating a sheet in the first place (if possible), instead of deleting it afterwards. If you do not want to tackle this, please open an issue for it.